fuzzers/jif/chromium_patches.diff

```diff
@@ -75,4 +75,4 @@ index 2a44aec23d..4ea02324d5 100644
 +     "//ui/gfx/geometry",
 +  ]
 +  libs = ["jif/libjif/target/release/libjif.a"]
-+ }
\ No newline at end of file
++ }
```

Comment:
This was actually preventing me from applying the patch on Linux. GNU patch doesn't support no endline.